packages/rancher-gatekeeper/rancher-gatekeeper.patch

```diff
@@ -1,26 +1,25 @@
 diff -x '*.tgz' -x '*.lock' -uNr packages/rancher-gatekeeper/charts-original/Chart.yaml packages/rancher-gatekeeper/charts/Chart.yaml
 --- packages/rancher-gatekeeper/charts-original/Chart.yaml
 +++ packages/rancher-gatekeeper/charts/Chart.yaml
-@@ -1,6 +1,6 @@
- apiVersion: v1
+@@ -3,8 +3,15 @@
  description: A Helm chart for Gatekeeper
```

Comment:
Can you make this a statement about what gatekeeper is, not what the chart is. We want to move away from description being `A helm chart....` as it displays in the UI.

Reply:
Updated